addons/pos_online_payment_self_order/tests/test_self_order_frontend.py

```diff
@@ -3,10 +3,16 @@
 from unittest.mock import patch
 
 import odoo.tests
+from odoo.addons.pos_restaurant.tests import test_frontend
 from odoo.addons.pos_self_order.tests.self_order_common_test import SelfOrderCommonTest
 # from odoo.addons.pos_online_payment.models.pos_payment_method import PosPaymentMethod
 
 
 @odoo.tests.tagged("post_install", "-at_install")
 class TestSelfOrderFrontendMobile(SelfOrderCommonTest):
     pass
+
+class TestSelfOrderOnlinePayment(test_frontend.TestFrontendCommon):
```

Comment:
Again, for consistency.
```suggestion
class TestSelfOrderOnlinePayment(TestFrontendCommon):
```